### DISCUSSION TITLE
agent cannot find any artifact with architecture arm64

### DISCUSSION THREAD

**Discussion by fregataa** [lablup/contribution-academy-2021#74] [Q&A] [answered]
## Error

when I run the command 
```shell
backend.ai run -c 'print("hello world!")' python
```
agent raises error which means it cannot find any distro.
the machine's architecutre is `arm64`.
I embedded codes to print all exec candidates and the below is the codes and the output.
> agent/src/ai/backend/agent/agent.py line-num 1130

![image](https://user-images.githubusercontent.com/44239739/130328368-c7cb7ec3-9e33-46df-b884-398f45d6ea50.png)

![image](https://user-images.githubusercontent.com/44239739/130327621-c35297bc-75eb-4c52-93e9-3cc31d0764ba.png)

it shows that there is no candidate with `arm64`.

### Environment Description
OS: macOS Big Sur 11.5.2
App version: 21.03.5
Python version: 3.9.5
Current Shell: zsh

## Expected output of agent

![image](https://user-images.githubusercontent.com/44239739/130327778-60e7f5db-7aea-4b45-9a4f-8f36cdb9e4b0.png)
this is expected output of agent with `x86_64` machine's architecture.

@euni00 @yujung7768903

**Comment by inureyes** [comment, marked as answer]
짐작컨대 지금 M1맥에 설치중이시라, 아직 테스트중인 ARM64 설치 코드를 타는거 같습니다. AWS Graviton 대응이 이번주 목요일에야 드래프트가 나와서 그 코드가 메인 브렌치에 반영되기 전이라서 위와 같은 에러가 나오는거겠습니다. 

ARM64기반인 M1맥에서의 동작 대응은 다음주나 다다음주쯤 되지 않을까 싶습니다.
